src/Common/Core/Impl/Logging/Implementation/ServiceLogger.cs

```diff
@@ -7,12 +7,16 @@
 using Microsoft.Common.Core.Disposables;
 using Microsoft.Extensions.Logging;
 
-namespace Microsoft.R.Host.UserProfile {
-    internal sealed class ServiceLogger : ILogger, IDisposable {
+namespace Microsoft.Common.Core.Logging {
+    public sealed class ServiceLogger : ILogger, IDisposable {
         private readonly string _category;
+        private readonly LogLevel _logLevel;
+        private readonly string _eventLogSource;
 
-        public ServiceLogger(string category) {
+        public ServiceLogger(string category, LogLevel minLogLevel, string eventLogSource) {
```

Comment:
I think service logger is never 'None' and generally always 'Minimal'. It is not controlled by user settings